tests/studio/test_equalizer.py

```diff
@@ -14,7 +14,14 @@
 
 
 def exact_comparator(recorded_result, playback_result):
-    return EqualityStatus.Equal if recorded_result == playback_result else EqualityStatus.Different
+    if type(recorded_result) != type(playback_result):
+        return EqualityStatus.Different
+    diff = abs(recorded_result - playback_result)
+    if diff == 0:
+        return EqualityStatus.Equal
+    if diff <= 10:
```

Comment:
Probably better to manage it by a percent. diff * 100 / playback_result <= 10 (or other number)

Reply:
It is just mock for tests purpose. I am engineering the case exactly as I need so not much value in investing here